test/modules/aws-vpc-integration.ts

```diff
@@ -151,7 +152,7 @@ describe('VPC Integration Testing', () => {
       INSERT INTO nat_gateway (connectivity_type, subnet_id, tags)
       SELECT 'private', id, '{"Name":"${ng}"}'
       FROM subnet
-      WHERE cidr_block = '192.${randIPBlock}.0.0/16';
+      WHERE cidr_block = '191.${randIPBlock}.0.0/16';
```

Comment:
why this change?

Reply:
For the new tests that update the cidr value further on, I believe. Changing from 191 to 192

Reply:
It is also because the previous tests were kind of wrong. They were leaving behind the vpc with the 192 cidr and that's why the rest of the tests where working, but technically they should have failed